### PR TITLE
Add Fig as an autocomplete method

### PR DESCRIPTION
**Pull request by brendanfalk**
### Proposed changes

[Fig](https://fig.io) provides autocomplete for 300+ CLI tools including Docker, so we would love to have Fig listed as an autocomplete method in your docs. Thanks so much and please let me know if you have any questions!

### CODE REVIEW

**Comment by dockertopia** [comment]
@brendanfalk thanks for the suggestion, let me check with the team. Could you tell how you got to this page?

**Comment by brendanfalk** [comment]
@dockertopia thanks for commenting! Docker is one of the most-used commands by Fig users, would love to see Fig listed among the autocomplete options.
image
I got to this page by clicking "edit this page" on the docs online.

File: compose/completion.md [hunk 1]
```diff
@@ -9,6 +9,16 @@ for the bash and zsh shell.
 
 ## Install command completion
 
+### Fig
+
+<a href="https://fig.io/" target="_blank"><img src="https://fig.io/badges/Logo.svg" width="15" height="15"/></a> [Fig](https://fig.io/) provides IDE-style autocompletions for Docker. It works in bash, zsh, and fish.
```
Comment (thaJeztah) [inline compose/completion.md:14]: Could you remove the hot-linked logo, and add a `target` and `rel=noopened` on the line to the Fig website?

```suggestion
[Fig](https://fig.io/){:target="_blank" rel="noopener"} provides IDE-style autocompletions for Docker. It works in bash, zsh, and fish.
```

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Hi @brendanfalk :wave: ! 

Thanks for contributing, and sorry for the delay on this PR. There was a bit of discussion internally about including a 3rd party solution in this section; Fig is great, and I know some people at Docker use it on their machines, but the question was if putting it here would become too much of an endorsement, and if it would become a precedent for _other_ possible solutions to also have to be included. Dora and I were trying to find the right person at Docker to give their "ok" for that, but that got a bit on the backlog; let me try again to do so, and get back to this PR when we get an answer!

Pending that, I left a suggestion on the changes itself 😅

File: compose/completion.md (hunk 1 above)
Comment (dockertopia) [reply compose/completion.md:14]: @thaJeztah Thanks so much for having a look at this. 
We might recover this page together with this feature in the upcoming future. For now, in line with the decision about the completion page, I'm closing this PR instead of leaving it open as there's no clear timeline. 
@brendanfalk thanks for your contribution, feel free to reopen this PR if this feature becomes available again in the near future.

**Comment by dockertopia** [comment]
See https://github.com/docker/docker.github.io/pull/15217#discussion_r951533132.

**Comment by dockertopia** [comment]
See https://github.com/docker/docker.github.io/pull/15217#discussion_r951533132.

File: compose/completion.md (hunk 1 above)
Comment (brendanfalk) [reply compose/completion.md:14]: Hey! I'm so incredibly sorry for the massive delays here. I completely missed the above comments. 

Very happy to merge with those changes by @thaJeztah above. Would this enable Fig to be merged?

> For now, in line with the decision about the completion page, I'm closing this PR instead of leaving it open as there's no clear timeline.

Not sure I fully follow this. Was there a decision somewhere else that I missed? Or is it the lack-of a final decision that is causing it to be closed?

Comment (brendanfalk) [reply compose/completion.md:14]: Ahh. My apologies. I now see the completions.md file has been removed. Makes total sense.

Thanks for the commentary. We will keep a close eye on things and be sure to contribute if/when things change again!
